### DISCUSSION TITLE
Sunbird RC - C4GT Projects

### DISCUSSION THREAD

**Discussion by sukhpreetssekhon** [Sunbird-RC/community#663] [Ideas & Enhancements]
@tejash-jl @Renuka-S - Please can you add and share the projects that are being put forward for Code 4 GovTech 2023. Thanks

**Comment by parthlawate** [comment]
Some Suggestions :

1. Admin Frontend to Configure Registry schema 
2. Admin Frontend to configure form UI aspects (Display in multiple columns, ordering and so on) (1 & 2 can be combined too) 
3. Web Installer for Sunbird RC

**Reply by sukhpreetssekhon** [reply]
Thanks Parth.

@tejash-jl - can you finalize and submit the projects for C4GT [here](https://forms.gle/FeGH7TTYc53b9wfV7)?
